### PR TITLE
(feat): Adding ansible custom lint

### PR DESCRIPTION
**Pull request by rajdas98**
Signed-off-by: Raj <raj.das@mayadata.io>

Addind custom ansible lint to check the ansible playbook
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [x] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: ansiblelint/rules/RestrictCommandsCheck.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from ansiblelint import AnsibleLintRule
+
+class RestrictCommandsCheck(AnsibleLintRule):
+  id = '702'
+  shortdesc = 'commands like (awk/cut/set) should not be use'
+  description = 'Certain commands like awk/cut/set should not be use'
```
Comment (ksatchit) [inline ansiblelint/rules/RestrictCommandsCheck.py]: Warn against usage of bash utilities like aw/sed/cut/grep

File: ansiblelint/rules/ShellModuleScalarRule.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from ansiblelint import AnsibleLintRule
+
+class ShellModuleScalarRule(AnsibleLintRule):
+  id = '700'
+  shortdesc = 'Scalar(">") is missing in shell command'
+  description = 'Required scalar if shell commands are greater than three'
```
Comment (ksatchit) [inline ansiblelint/rules/ShellModuleScalarRule.py]: Use scalar for shell commands with multiple args

Comment (rajdas98) [reply ansiblelint/rules/ShellModuleScalarRule.py]: Done

File: ansiblelint/rules/RestrictCommandsCheck.py (hunk 1 above)
Comment (rajdas98) [reply ansiblelint/rules/RestrictCommandsCheck.py]: Done

File: ansiblelint/rules/RestrictCommandsCheck.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+from ansiblelint import AnsibleLintRule
+
+class RestrictCommandsCheck(AnsibleLintRule):
+  id = '702'
+  shortdesc = 'Warn against usage of bash utilities like awk/sed/cut/grep'
+  description = 'Certain commands like awk/sed/cut/grep should not be use'
```
Comment (ksatchit) [inline ansiblelint/rules/RestrictCommandsCheck.py]: ...It is recommended to use ansible/python native commands against bash utils like awk/sed/cut/grep

File: ansiblelint/rules/ShellModuleScalarRule.py (hunk 2 above)
Comment (ksatchit) [reply ansiblelint/rules/ShellModuleScalarRule.py]: Requires scalar if shell command consist of args greater than three

File: ansiblelint/rules/RestrictCommandsCheck.py (hunk 3 above)
Comment (rajdas98) [reply ansiblelint/rules/RestrictCommandsCheck.py]: done

File: ansiblelint/rules/ShellModuleScalarRule.py (hunk 2 above)
Comment (rajdas98) [reply ansiblelint/rules/ShellModuleScalarRule.py]: done

**Pull request merged**